cel/options.go

```diff
@@ -541,22 +563,58 @@ func configToEnvOptions(config *env.Config, provider types.Provider, optFactorie
 		}
 		envOpts = append(envOpts, FunctionDecls(funcs...))
 	}
-	for _, e := range config.Extensions {
-		extHandled := false
-		for _, optFac := range optFactories {
-			if opt, useOption := optFac(e); useOption {
-				envOpts = append(envOpts, opt)
-				extHandled = true
-				break
-			}
+
+	// Configure features
+	for _, feat := range config.Features {
+		if id, found := featureIDByName(feat.Name); found {
+			envOpts = append(envOpts, features(id, feat.Enabled))
```

Comment:
For any misconfigured features (e.g: typo), do you want to surface this back to the uesr?

Reply:
I know, I thought about that, but I think I need to skip the feature since it could not be supported on a specific CEL stack. I can come back to this, but I couldn't think of a good way to mark a features as "intended for support in config files".